recipes/libcap/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+import glob
+
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibcapConan(ConanFile):
+    name = "libcap"
+    license = ("GPL-2.0", "BSD-3-Clause")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://git.kernel.org/pub/scm/libs/libcap/libcap.git"
+    description = "This is a library for getting and setting POSIX.1e" \
+                  " (formerly POSIX 6) draft 15 capabilities"
+    topics = ("conan", "libcap", "capabilities")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "psx_syscals": [True, False]}
+    default_options = {'shared': False, "psx_syscals": False}
+    _autotools = None
+    _autotools_env = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Only Linux supported")
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        src_folder = "{}-{}".format(self.name, self.version)
+        os.rename(src_folder, self._source_subfolder)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools, self._autotools_env
+        self._autotools = AutoToolsBuildEnvironment(self)
+        self._autotools_env = self._autotools.vars
+        self._autotools_env["SHARED"] = \
+            "yes" if self.options.shared else "no"
+        self._autotools_env["PTHREADS"] = \
+            "yes" if self.options.psx_syscals else "no"
+        self._autotools_env["DESTDIR"] = self.package_folder
+        self._autotools_env["prefix"] = "/"
+        self._autotools_env["lib"] = "lib"
+        return self._autotools, self._autotools_env
+
+    def build(self):
+        with tools.chdir(os.path.join(self._source_subfolder, self.name)):
+            env_build, env_build_vars = self._configure_autotools()
+            env_build.make(vars=env_build_vars)
+
+    def package(self):
+        self.copy("License", dst="licenses", src=self._source_subfolder)
+
+        with tools.chdir(os.path.join(self._source_subfolder, self.name)):
+            env_build, env_build_vars = self._configure_autotools()
+            env_build.install(vars=env_build_vars)
```

Comment:
I just skimmed through the Makefile at the cgit browser at https://git.kernel.org/pub/scm/libs/libcap/libcap.git/tree/libcap/Makefile#n124
BUut it looks like there exists 2 interesting install targets: `install-static` and`install-shared` ( installing static/shared libraries.
`install-common` can be used for the rest.

Reply:
Yes, it works. Also it allows to remove unused header file from package in some cases. Thanks.